src/Orc.LicenseManager.Client.WPF/Orc.LicenseManager.Client.WPF.Shared/Views/NetworkLicenseUsageWindow.xaml

```diff
@@ -21,14 +21,14 @@
         <StackPanel>
             <StackPanel>
                 <TextBlock Margin="10" TextWrapping="Wrap" FontSize="16" 
-                       Text="It appears that you have reached the maximum number of concurrent usages of the license."/>
+                       Text="{catel:LanguageBinding LicenseManager_NetworkLicenseUsageWindow_TextBlock_Text_ItAppearsThatYouHaveReachedTheMaximumNumberOfConcurrentUsagesOfTheLicense}"/>
                 <GridSplitter Style="{StaticResource HorizontalGridSplitterStyle}" IsEnabled="False"/>
             </StackPanel>
             
             <TextBlock Margin="10" TextWrapping="Wrap"
                        Text="{Binding MaximumNumberOfConcurrentUsages, StringFormat='You are entitled to {0} concurrent usages of the license, but it looks this amount is already used.'}"/>
```

Comment:
Please convert this to a converter instead of a stringformat and load the appropriate resource.